src/montgomery.rs

```diff
@@ -65,7 +91,7 @@ impl CompressedMontgomeryU {
     /// * `v` is not square.
     //
     // XXX any other exceptional points for the birational map?
-    pub fn decompress(&self) -> Option<ExtendedPoint> {
+    pub fn decompress_edwards(&self) -> Option<ExtendedPoint> {
```

Comment:
I don't remember where we'd actually want to use this, instead of just lifting a MontgomeryPoint to an ExtendedPoint... Should we keep it around?

Reply:
I kept it around because I wasn't yet fully convinced that the conversion equations we worked out on the board were correct and non-exceptional. I have a git stash where I was working on it, but since the only benefit of that would be speed, I didn't see a reason to remove this until the less kludgy thing is finished properly. Do you feel like Montgomery P² ↦ Edwards P³ conversion would be a blocker for 1.0.0?